### PR TITLE
Feature/pretifying wizard trng 1104

### PR DESCRIPTION
**Pull request by tomwim**
### Description
The Training Setup Wizard now uses radio buttons instead of check boxes.
In the toolbar the submenu 'Creator" was removed and all items are now on the top level under 'Innoactive'. Additionally, 'Windows' was renamed to 'Settings'.

### Type of change
- [X] This change requires a documentation update

### Aditional Notes
This change would require a documentation update where to find everything in the toolbar but as discussed in a meeting users should still be able to follow the documentation with the new slightly changed structure. As soon as we move the Creator into 'Tools' we will do a bigger documentation update.

Split into two commits:
1. Radio Buttons
2. Toolbar changes

### CODE REVIEW

**Review by Gusinuhe — DISMISSED** [review]

**Review by SimonTheSourcerer — APPROVED** [review]

**Comment by tomwim** [comment]
Ready for re-review. Max changed some help/warning text in the Creator which I did not see and I fixed the toggles to only toggle on in the Training Scene Setup Page

**Review by Maxsiom — APPROVED** [review]

**Comment by tomwim** [comment]
Waiting for an approval by @SimonTheSourcerer and then I'm going to merge

File: Editor/UI/CreatorEditorStyles.cs
Comment (aleksei-korolev) [inline Editor/UI/CreatorEditorStyles.cs:117]: Can't we move it to default value declaration or static constructor?

**Review by aleksei-korolev — APPROVED** [review]

**Review by SimonTheSourcerer — APPROVED** [review]

**Pull request merged**
